core/feconf.py

```diff
@@ -106,6 +107,43 @@
 LEGACY_HTML_FIELD_TYPES_TO_RULE_SPECS_EXTENSIONS_MODULE_DIR = os.path.join(
     'interactions', 'legacy_html_field_types_to_rule_specs_by_state_version')
 
+
+# Valid model names.
+# TODO(#14419): Change naming style of Enum class from SCREAMING_SNAKE_CASE
+# to PascalCase and its values to UPPER_CASE. Because we want to be consistent
+# throughout the codebase according to the coding style guide.
+# https://github.com/oppia/oppia/wiki/Coding-style-guide
+class NAMES(enum.Enum): # pylint: disable=invalid-name
```

Comment:
I think you'll need a more descriptive name than this, given that this constant is in a file with lots of other constants.

Reply:
How abt `VALID_MODEL_NAMES`